scripts/template/src/patternfly/{folder}/{name}/{nameDasherized}.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="{{bemName}}{{bemModifierBlock}}"
+<div class="{nameBEM}{{bemModifierBlock}}"
```

Comment:
This outputs 

```
<div class="pf-c-foo-bar{{bemModifierBlock}}"
  {{attributeBlock}}>
  {{partialBlock}}
</div>
```

Instead of

```
<div class="pf-c-foo-bar{{#if foo-bar--modifier}} {{foo-bar--modifier}}{{/if}}"
  {{#if foo-bar--attribute}}
    {{foo-bar--attribute}}
  {{/if}}>
  {{>@partial-block}}
</div>
```

Reply:
Fixed.